pandas/tests/io/test_pickle.py

```diff
@@ -284,14 +286,30 @@ def get_random_path():
     return f"__{tm.rands(10)}__.pickle"
 
 
+@pytest.fixture(
```

Comment:
can these share with the same fixture you defined for test_common? can you just move them to a higher level?